Source/inv.cpp

```diff
@@ -1928,14 +1928,14 @@ void AutoGetItem(int pnum, ItemStruct *item, int ii)
 	plr[pnum].HoldItem._itype = ITYPE_NONE;
 }
 
-int FindGetItem(int idx, WORD ci, int iseed)
+auto FindGetItem(int idx, WORD ci, int iseed) -> int

```

Comment:
what is this monstrosity?

Reply:
That's a valid syntax, but less readable in that case. Not sure why clang-tidy has decided to make that change.

Reply:
I have no clue, clang-tidy appeared to want to do this change to everything so I dropped all of them but missed this one. @glebm why is this smart? It looks like the same thing, but with more words and symbols with just makes it more cryptic?

Reply:
Trailing return types are usually used for lambdas or templated functions where the return type depends on the template arguments. It makes no sense here.

Maybe there's some new-age movement to "trailing return all the things!", like there is an "auto all the things!" movement.

Reply:
This check is disabled in the config I submitted. Perhaps you've enabled it accidentally?

Reply:
Hmm, I expected it to be disabled with how in ran individual checks, but guess not. Will try and avoid that for the next run of 1-rule-all-files (pending feedback on an all-rules-few-files run https://github.com/diasurgical/devilutionX/pull/1623).